arangod/RocksDBEngine/RocksDBEngine.h

```diff
@@ -397,6 +399,7 @@ class RocksDBEngine final : public StorageEngine {
 
   void trackRevisionTreeBufferedMemoryIncrease(std::uint64_t value) noexcept;
   void trackRevisionTreeBufferedMemoryDecrease(std::uint64_t value) noexcept;
+  uint64_t treeBufferedMemoryUsage() const noexcept;
```

Comment:
I added this only for testing. Currently not called from anywhere, so can be removed again if required.
```suggestion
```